layouts/shortcodes/security-products/suppressions-granular-access.md

```diff
@@ -0,0 +1,17 @@
+By default, all users have full access to [suppressions][201]. To use granular access controls to limit the roles that may edit a suppression rule:
+
+1. Click the vertical three-dot menu for the rule and select **Permissions**.
+1. Click **Restrict Access**. The dialog box updates to show that members of your organization have **Viewer** access by default.
```

Comment:
Same question as above re: what "by default" means in this context.

Reply:
So when you click Restrict Access, it shows that your organization (eg. Datadog Demo (11287)) is a Viewer in the dropdown menu. I'll combine the next step and this one, because the next step mentions the dropdown menu.